src/styles/globals.css

```diff
@@ -107,3 +112,26 @@ a {
     color-scheme: dark;
   }
 }
+
+h1 {
+  font-family: "Audiowide" !important;
+  font-weight: 500;
+  color: black;
+}
+
+h2 {
+  font-family: "Audiowide" !important;
+  font-weight: 400;
+  color: black;
+}
+
+h3 {
+  font-family: "Lato" !important;
+  font-weight: 300;
+  color: black;
+}
+
+
+p {
+  font-family: "Lato", sans-serif;
+}
```

Comment:
Great job styling these in global.css! In the screenshot below, I have an example of the h1, h2, h3, and p as styled currently. I think we can increase the font size on h1 and p, and the font weight on p--could we match these to the Figma?
<img width="478" alt="image" src="https://github.com/IllinoisWCS/IllinoisWCS.github.io/assets/80798381/b446aa29-ad86-406c-be6f-0d2fbbace078">